packages/blade/src/components/Tag/Tag.tsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Box } from '~components/Box';
+import type { StyledPropsBlade } from '~components/Box/styledProps';
+import { getStyledProps } from '~components/Box/styledProps';
+import type { IconButtonProps } from '~components/Button/IconButton';
+import { IconButton } from '~components/Button/IconButton';
+import type { IconComponent, IconProps } from '~components/Icons';
+import { CloseIcon } from '~components/Icons';
+import type { TextProps } from '~components/Typography';
+import { Text } from '~components/Typography';
+import { metaAttribute, MetaConstants } from '~utils/metaAttribute';
+import type { StringChildrenType, TestID } from '~utils/types';
+import { isReactNative, makeSize } from '~utils';
+import { size as globalSizeTokens } from '~tokens/global';
+import BaseBox from '~components/Box/BaseBox';
+import type { PaddingProps } from '~components/Box/BaseBox/types/spacingTypes';
+
+type TagProps = {
+  /**
+   * Decides the size of Tag
+   *
+   * @default medium
+   */
+  size?: 'medium' | 'large';
+
+  /**
+   * Leading icon for your Tag
+   */
+  icon?: IconComponent;
+
+  /**
+   * Callback when close icon on Tag is clicked
+   */
+  onDismiss: () => void;
+
+  /**
+   * Text that renders inside Tag
+   */
+  children: StringChildrenType;
+
+  /**
+   * Disable tag
+   */
+  isDisabled?: boolean;
+
+  /**
+   *
```

Comment:
Should we mention that this is an internal only tag and shouldn't be used in the jsdoc?

Reply:
done